dat/missions/shark/sh07_pirates.lua

```diff
@@ -274,3 +281,7 @@ function generic_dead()
       mem.marker2 = misn.markerAdd(paypla, "low")
    end
 end
+
+function generic_jumped()
+   tk.msg(_("Target is gone!"), fmt.f(_("It seems the pirate left. Don't worry: I'll bet that if you come back a little later, they will be back.")
```

Comment:
No need to use `fmt.f` if you aren't adding special strings with `{foo}`.

Reply:
Thanks for the catch which also prompted be to check my brackets... I missed two closing ones! :( clearly not on my game this morning! - fixed now and also fixed some formatting consistancy with the previous function; the [[s & ]]s is this good?